contents/docs/web-analytics/getting-started.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Getting started with web analytics
+sidebar: Docs
+showTitle: true
+---
+
+It's easy to get up and running with [web analytics](/web-analytics). All you need to do is [install PostHog](/docs/getting-started/install) and you'll automatically start collecting data based on traffic to your app or website.
+
+Web analytics provides you with a simple [dashboard](/docs/web-analytics/dashboard), but it can be overwhelming if you don't have a technical background or experience with marketing concepts.
```

Comment:
Im not sure we should frame it as "`if you don't have a technical background`, since in general we assume that our users are technical. My concern is that an engineer would read this line and think this doc is not for them. So maybe we should just have "if you're new to marketing/web analytics bit"

Reply:
Fair point. I think I picked up this article from the mega issue and it literally said "(for non-technical people)", but you're right if we're writing with the assumption that all our users/readers are technical. I went ahead and tweaked the wording and removed the "overwhelming" bit.